src/store/actions/actions.js

```diff
@@ -300,12 +295,22 @@ export const loginUser = token => {
   };
 };
 
-export const logoutUser = () => dispatch => {
+export const getUserForms = (token) => {
+  const reqConfig = {
+    token,
+    endpoint: 'forms',
+  };
+  // eslint-disable-next-line
+  return dispatch => fetchAPIForms(reqConfig, dispatch);
+};
```

Comment:
@ciremusyoka - Why were these actions updated? (7d9436ed4247e138b2ee880144b4a495aeb5fc37)

Reply:
@cKellyDesign - Had to rebase with staging to get the latest code version for the type definitions to be up date.